### PR TITLE
Update 'failed_when_contains' error messages.

### PR DESCRIPTION
**Pull request by AlexLardschneider**
# Description

The following document by HUAWEI lists the most common error messages of HUAWEI VRP:
https://support.huawei.com/enterprise/en/doc/EDOC1000177810/46c4f52c/interpreting-command-line-error-messages

However, since I found that there are many more (20+) different error messages not listed in this document, and some even vary slightly by VRP version, I think it's better to only match `Error:`. 

This PR replaces the more specific error messages with one generic `Error:` message, which will hopefully cover all the different cases.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] This change requires a documentation update

# How Has This Been Tested?

Tested on a couple of different devices running VRP by purposely forcing errors.

# Checklist:

- [X] My code follows the style guidelines of this project (no GitHub actions compalints! run `make lint` before
 committing!)
- [X] I have commented my code, pydocstyle and darglint are happy, docstrings are in google docstring format, and all
 docstrings include a summary, args, returns and raises fields (even if N/A)
- [ ] I have added tests that prove my fix is effective or that my feature works, if adding "functional" tests please
 note if there are any considerations for the vrnetlab setup
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: scrapli_community/huawei/vrp/huawei_vrp.py [hunk 1]
```diff
@@ -57,9 +57,7 @@
         "sync_on_close": default_sync_on_close,
         "async_on_close": default_async_on_close,
         "failed_when_contains": [
-            "Error: Unrecognized command found at '^' position.",
-            "Error: Wrong parameter found at '^' position.",
-            "Error:Incomplete command found at '^' position.",
```
Comment (carlmontanari) [inline scrapli_community/huawei/vrp/huawei_vrp.py:62]: missing a space between "Error:" and "Incomplete" maybe?

Comment (AlexLardschneider) [reply scrapli_community/huawei/vrp/huawei_vrp.py:62]: This is actually one of the things that differ from one VRP version to the other. 
Some versions/devices output the error with a space between the ":" and the actual message, and some without.

Comment (AlexLardschneider) [reply scrapli_community/huawei/vrp/huawei_vrp.py:62]: One of the main reasons why I decided to only match `Error:` instead of the full error messages.

Comment (carlmontanari) [reply scrapli_community/huawei/vrp/huawei_vrp.py:62]: lol im an idiot :) was looking at the old side vs the new side.... carry on sir :D lol

Comment (AlexLardschneider) [reply scrapli_community/huawei/vrp/huawei_vrp.py:62]: Haha, no worries! :)

**Pull request merged**